bundle/run/job.go

```diff
@@ -235,6 +235,9 @@ func (r *jobRunner) Run(ctx context.Context, opts *Options) (output.RunOutput, e
 	case jobs.RunResultStateTimedout:
 		log.Infof(ctx, "Run has timed out!")
 		return nil, fmt.Errorf("run timed out: %s", run.State.StateMessage)
+
+	// TODO: handle other result states.
+	default:
```

Comment:
There are a few more states we need to handle. This is the only real problem that this linter surfaced.